policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/applicable_amount.yaml

```diff
@@ -79,21 +63,25 @@ SURVIVING_SPOUSE:
     2021-01-01: 334_150
     2022-01-01: 343_950
     2023-01-01: 368_900
-  uprating:
-    parameter: gov.irs.uprating
-    rounding:
-      type: downwards
-      interval: 50
+    2024-01-01: 388_400
 metadata:
   label: Virginia itemized deduction applicable amount
   period: year
   unit: currency-USD
   breakdown:
   - filing_status
+  propagate_metadata_to_children: true
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
   reference:
-    # Virginia continued to increase the federal applicable amount with inflation 
+    # Virginia's limit conforms with the federal limit. The federal limit increases with inflation; thus, so does Virginia's.
```

Comment:
as in it equals the federal parameter? if so can we just point to that and skip this parameter? please cite virginia statute

Reply:
Virginia continued the pease limitation despite it being repealed under TCJA - this replicates the value which we set to 0 for those years [here](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/irs/deductions/itemized/reduction/agi_threshold.yaml)